autogen/main/variables.tf.tmpl

```diff
@@ -703,12 +707,6 @@ variable "kalm_config" {
   default     = false
 }
 
-variable "config_connector" {
```

Comment:
If we care about minimizing the diff in the beta modules, I can move it back here and add additional `if` blocks